Resources/Private/Templates/Backend/Guest.html

```diff
@@ -1,6 +1,6 @@
 <f:section name="guestFrameApplication">
     <script>window.neos = window.parent.neos;</script>
     <script src="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/Plugins/ckeditor/ckeditor.js')}"></script>
-    <script src="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/JavaScript/Guest.js')}"></script>
-    <link rel="stylesheet" href="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/Styles/Guest.css')}">
+    <script src="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/Javascript/Guest.js')}"></script>
+    <link rel="stylesheet" href="{f:uri.resource(path: 'resource://Neos.Neos.Ui/Public/Styles/Host.css')}">
```

Comment:
Maybe call them Host.css, Shared.css and Guest.css? Current naming is confusing, as I would expect `Host.css` to be only loaded for Host, based on its name.